Increase buffer size for IoT agent
---
--- a/src/agent/wra_h.h   2015-08-24 15:57:17.056745822 +0200
+++ b/src/agent/wra_h.h   2015-08-24 15:57:01.016746297 +0200
@@ -69,7 +69,7 @@
 #define WRA_TRUE 1
 #define WRA_FALSE 0
 #define WRA_TM_MAX_NAME_LEN 128
-#define WRA_TM_MAX_STR_VALUE_LEN 256 
+#define WRA_TM_MAX_STR_VALUE_LEN 1024 
 #define WRA_TM_MAX_DESC_LEN 256 
 #define WRA_AP_EX_APPNAME_LEN   32
 #define WRA_SUBSCRIBE_NAME_LEN   32
